pkg/repo/repo.go

```diff
@@ -138,6 +139,34 @@ func GetMetaFromStore(msg []byte, name string) (interface{}, error) {
 	return meta, nil
 }
 
+// TODO make public in cosign -- we need to wait for go-tuf changes.
+type customMetadata struct {
+	Usage  string `json:"usage"`
```

Comment:
Instead of string, can these by `UsageKind` and `StatusKind`? I think we can pull in those from the TUF package in Cosign.

Reply:
Done! Had to change Cosign UsageKind to Unknown, but that's fine -- no one consumes it in code